crates/cargo-wdk/tests/test_utils/mod.rs

```diff
@@ -242,6 +242,7 @@ pub fn create_cargo_wdk_cmd<P: AsRef<Path>>(
     cmd_name: &str,
     cmd_args: Option<&[&str]>,
     curr_working_dir: Option<P>,
+    env_vars: Option<&[(&str, Option<&str>)]>,
```

Comment:
Nit: move `env_vars` right after `cmd_args`. They are better organized that way.

Reply:
Done